docs/sources/tempo/metrics-from-traces/metrics-queries/sampling-guide.md

```diff
@@ -0,0 +1,528 @@
+---
+title: TraceQL metrics sampling guide
+menuTitle: Sampling guide
+description: Optimize TraceQL metrics query performance using sampling hints
+weight: 500
+keywords:
+  - TraceQL metrics
+  - sampling
+  - performance optimization
+  - query optimization
+---
+
+# TraceQL metrics sampling guide
+
+{{< docs/shared source="tempo" lookup="traceql-metrics-admonition.md" version="<TEMPO_VERSION>" >}}
+
+TraceQL metrics sampling is a performance optimization feature that enables faster query execution by processing a subset of trace data while maintaining acceptable accuracy. This guide provides essential information for implementing sampling in your TraceQL metrics queries.
+
+## Introduction and overview
+
+TraceQL metrics sampling addresses the challenge of balancing query performance with data accuracy when working with large-scale trace datasets. Traditional TraceQL metrics queries can be resource-intensive, especially when aggregating data across millions of spans or thousands of traces over extended time periods.
+
+Sampling intelligently selects a representative subset of data for processing, delivering 2-4x performance improvements for heavy aggregation queries while maintaining 95%+ accuracy for most use cases. This makes sampling particularly valuable for:
+
+- **Real-time dashboards** requiring fast refresh rates
+- **Exploratory data analysis** where approximate results accelerate insights
+- **Resource-constrained environments** with limited compute capacity
+- **Large-scale deployments** processing terabytes of trace data daily
+
+Grafana Tempo supports three complementary sampling approaches: adaptive sampling that automatically optimizes based on query characteristics, fixed span-level sampling for precise control over span selection, and trace-level sampling optimized for trace-wide aggregations.
```

Comment:
This doesn't sound right -- seems out of scope for just the TraceQL metrics sampling as opposed to sampling in general.